.gitignore

```diff
@@ -53,6 +53,7 @@ cpu.out
 /bin
 /dist
 /custom/*
+!/custom/conf
```

Comment:
`!/custom/conf` should be ignored because it contains local "app.ini"

Reply:
without this line unfortunately `git` does not commit changes for the specific file listed below, I guess because the above `/custom/*` has a `*`... that is just my guess. May be worth then ignoring specifically `/custom/conf/app.ini` and not just `/custom/*`?

Reply:
> without this line unfortunately git does not commit changes for the specific file listed below

Which file?

TBH I don't see a reason why it needs to add "!/custom/conf" into the list.

Reply:
<details>
<summary>hm, I have tried to reproduce my (well -- datalad) problem committing those changes and failed</summary> 

```shell
❯ git clone https://github.com/go-gitea/gitea/
Cloning into 'gitea'...
remote: Enumerating objects: 228164, done.
remote: Counting objects: 100% (1393/1393), done.
remote: Compressing objects: 100% (1057/1057), done.
^Cceiving objects:   9% (21905/228164), 9.76 MiB | 3.88 MiB/s
❯ git clone https://github.com/go-gitea/gitea/
❯ rm -rf gitea
❯ git clone --depth 1 https://github.com/go-gitea/gitea/
Cloning into 'gitea'...
remote: Enumerating objects: 6266, done.
remote: Counting objects: 100% (6266/6266), done.
remote: Compressing objects: 100% (5456/5456), done.
remote: Total 6266 (delta 602), reused 2877 (delta 380), pack-reused 0
Receiving objects: 100% (6266/6266), 10.02 MiB | 5.14 MiB/s, done.
Resolving deltas: 100% (602/602), done.
❯ cd gitea
BSDmakefile          Makefile      custom/   package-lock.json     snap/
CHANGELOG.md         README.md     docker/   package.json          templates/
CODE_OF_CONDUCT.md   README_ZH.md  docs/     playwright.config.js  tests/
CONTRIBUTING.md      SECURITY.md   go.mod    poetry.lock           vitest.config.js
DCO                  assets/       go.sum    poetry.toml           web_src/
Dockerfile           build/        main.go   public/               webpack.config.js
Dockerfile.rootless  build.go      models/   pyproject.toml
LICENSE              cmd/          modules/  routers/
MAINTAINERS          contrib/      options/  services/
❯ echo change >> ./custom/conf/app.example.ini
changes on filesystem:                                                                              
 custom/conf/app.example.ini | 1 +
❯ git commit -m 'Can we commit?' ./custom/conf/app.example.ini
[main ee3c3bd] Can we commit?
 1 file changed, 1 insertion(+)
❯ git reset HEAD^
Unstaged changes after reset:
M	custom/conf/app.example.ini
changes on filesystem:                                                                              
 custom/conf/app.example.ini | 1 +
❯ git commit -m 'Can we commit?' -a
[main f91638a] Can we commit?
 1 file changed, 1 insertion(+)

```
</details>

so I am not sure why exactly `git` was refusing to commit for me yesterday any longer.

I also realized that I ended up skipping fixing that `ALLWAYS` (btw - was it intended to be that instead of "always"?) : https://github.com/go-gitea/gitea/blob/main/custom/conf/app.example.ini#L510

Reply:
the point is - I will skip this commit